authors.txt

```diff
@@ -245,7 +245,7 @@ for example using the following format:</p>
 
 <ul>
 <li>W. Hart, F. Johansson and S. Pancratz. FLINT: Fast Library for
-    Number Theory, 2013. version 2.4.0, <tt>http://flintlib.org</tt>.
+    Number Theory, 2013. version 2.4.0, <tt>https://flintlib.org</tt>.
```

Comment:
Presumably the version and date here should be updated, too, but I wanted to keep this PR focussed